From 79bcea56ac1c2e812014afbb8a624fa60d5a87d7 Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Thu, 7 Jan 2021 16:22:50 +0200
Subject: [PATCH 34/78] dts: bluebox3: enable LLCE

Add status="okay" for the LLCE interfaces used on Bluebox 3.

Issue: ALB-6274

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
---
 .../dts/freescale/fsl-s32g274a-bluebox3.dts   | 90 ++++++++++++++++++-
 1 file changed, 89 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a-bluebox3.dts b/arch/arm64/boot/dts/freescale/fsl-s32g274a-bluebox3.dts
index eb83bb1eb8f7..52c8be8cf282 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a-bluebox3.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a-bluebox3.dts
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 /*
- * Copyright 2020 NXP
+ * Copyright 2020-2021 NXP
  */
 
 /dts-v1/;
@@ -103,6 +103,94 @@
 	status = "okay";
 };
 
+&llce0_sram {
+	status = "okay";
+};
+
+&llce1_sram {
+	status = "okay";
+};
+
+&llce2_sram {
+	status = "okay";
+};
+
+&llce3_sram {
+	status = "okay";
+};
+
+&llce_shmem {
+	status = "okay";
+};
+
+&llce_boot_status {
+	status = "okay";
+};
+
+&llce_can0 {
+	status = "okay";
+};
+
+&llce_can1 {
+	status = "okay";
+};
+
+&llce_can2 {
+	status = "okay";
+};
+
+&llce_can3 {
+	status = "okay";
+};
+
+&llce_can4 {
+	status = "okay";
+};
+
+&llce_can5 {
+	status = "okay";
+};
+
+&llce_can6 {
+	status = "okay";
+};
+
+&llce_can7 {
+	status = "okay";
+};
+
+&llce_can8 {
+	status = "okay";
+};
+
+&llce_can9 {
+	status = "okay";
+};
+
+&llce_can10 {
+	status = "okay";
+};
+
+&llce_can11 {
+	status = "okay";
+};
+
+&llce_can12 {
+	status = "okay";
+};
+
+&llce_can13 {
+	status = "okay";
+};
+
+&llce_can14 {
+	status = "okay";
+};
+
+&llce_can15 {
+	status = "okay";
+};
+
 &pcie0 {
 	status = "okay";
 };
-- 
2.25.1

